src/lib.rs

```diff
@@ -28742,6 +28742,44 @@ mod tests {
     "#,
       "@property --property-name{syntax:\"<color>\";inherits:true;initial-value:#00f}.foo{color:var(--property-name)}",
     );
+
+    test(
+      r#"
+      @media (width < 800px) {
+        @property --property-name {
+          syntax: '*';
+          inherits: false;
+        }
+      }
+    "#,
+      indoc! {r#"
+        @media (width < 800px) {
+          @property --property-name {
+            syntax: "*";
+            inherits: false
+          }
+        }
+    "#},
+    );
+
+    test(
+      r#"
+      @layer foo {
+        @property --property-name {
+          syntax: '*';
+          inherits: false;
+        }
+      }
+    "#,
+      indoc! {r#"
+        @layer foo {
+          @property --property-name {
+            syntax: "*";
+            inherits: false
+          }
+        }
+    "#},
+    );
```

Comment:
According to the specification, it is recommended to include other at rules in the test.

e.g. 

- `@supports`
- `@scope`
- `@container`

https://github.com/web-platform-tests/wpt/blob/c18682f41f9cf95a3909d76b21aad274abe8914f/css/css-conditional/container-queries/container-inner-at-rules.html#L49-L55